Directory.Packages.props

```diff
@@ -0,0 +1,108 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>false</CentralPackageTransitivePinningEnabled>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="ArangoDBNetStandard" Version="2.0.1" />
+    <PackageVersion Include="AWSSDK.CloudWatch" Version="3.7.300.7" />
+    <PackageVersion Include="AWSSDK.DynamoDBv2" Version="3.7.300.7" />
+    <PackageVersion Include="AWSSDK.S3" Version="3.7.302" />
+    <PackageVersion Include="AWSSDK.SecretsManager" Version="3.7.300.7" />
+    <PackageVersion Include="AWSSDK.SimpleNotificationService" Version="3.7.300.7" />
+    <PackageVersion Include="AWSSDK.SimpleSystemsManagement" Version="3.7.301.3" />
+    <PackageVersion Include="AWSSDK.SQS" Version="3.7.300.7" />
+    <PackageVersion Include="Azure.Data.Tables" Version="12.8.2" />
+    <PackageVersion Include="Azure.DigitalTwins.Core" Version="1.4.0" />
+    <PackageVersion Include="Azure.Identity" Version="1.12.0" />
+    <PackageVersion Include="Azure.Messaging.EventHubs" Version="5.10.0" />
+    <PackageVersion Include="Azure.Messaging.ServiceBus" Version="7.17.0" />
+    <PackageVersion Include="Azure.Search.Documents" Version="11.5.0" />
+    <PackageVersion Include="Azure.Security.KeyVault.Certificates" Version="4.5.1" />
+    <PackageVersion Include="Azure.Security.KeyVault.Keys" Version="4.5.0" />
+    <PackageVersion Include="Azure.Security.KeyVault.Secrets" Version="4.5.0" />
+    <PackageVersion Include="Azure.Storage.Blobs" Version="12.19.1" />
+    <PackageVersion Include="Azure.Storage.Files.Shares" Version="12.17.1" />
+    <PackageVersion Include="Azure.Storage.Queues" Version="12.17.1" />
+    <PackageVersion Include="Confluent.Kafka" Version="2.3.0" />
+    <PackageVersion Include="coverlet.collector" Version="6.0.0" />
+    <PackageVersion Include="Dapr.Client" Version="1.12.0" />
+    <PackageVersion Include="DogStatsD-CSharp-Client" Version="8.0.0" />
+    <PackageVersion Include="Ductus.FluentDocker" Version="2.10.59" />
+    <PackageVersion Include="Elastic.Clients.Elasticsearch" Version="8.14.3" />
+    <PackageVersion Include="EventStore.Client" Version="22.0.0" />
+    <PackageVersion Include="EventStore.Client.Grpc.Streams" Version="23.1.0" />
+    <PackageVersion Include="Google.Cloud.Firestore" Version="3.4.0" />
+    <PackageVersion Include="Gremlin.Net" Version="3.7.0" />
+    <PackageVersion Include="Hangfire.Core" Version="1.8.6" />
+    <PackageVersion Include="IBMMQDotnetClient" Version="9.3.4" />
+    <PackageVersion Include="InfluxDB.Client" Version="4.14.0" />
+    <PackageVersion Include="KubernetesClient" Version="14.0.2" />
```

Comment:
This was 12.1.1 previously. I don't think we should upgrade it in this PR.

https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/508123eda94dd2a02a7260cef0c1562ae40d215e/src/HealthChecks.Kubernetes/HealthChecks.Kubernetes.csproj#L12

https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/508123eda94dd2a02a7260cef0c1562ae40d215e/src/HealthChecks.UI.K8s.Operator/HealthChecks.UI.K8s.Operator.csproj#L14

Reply:
This version came from the `HealthChecks.UI` package.

see this https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/commit/eec153e994a472808d2dd9a94b046a7cc6e36d93

Reply:
Right- this is one of the drawbacks/things to be aware of when using CPM - all the packages need to have the same dependency version.

IMO - this change shouldn't be updating the dependency version in the above 2 projects. Those should use `PackageOverride=12.1.1` to get back to the previous version. Then we should log an issue and make a separate PR to update these versions for those 2 packages.